.changeset/wild-jokes-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': minor
+---
+
+Updated to support deployments of Azure DevOps Server under \tfs or similar subpath
```

Comment:
```suggestion
Updated to support deployments of Azure DevOps Server under TFS or similar subpath
```

Reply:
@Rugvip Sorry, I think we missed the backticks - `\tfs`. This is actually a folder, it used to be that when you installed TFS (the old name for Azure DevOps Server) it would end up like this: `http://mydomain.com/tfs`. But I think I also provided the wrong slash, it should probably be `/tfs`?